cobalt/watchdog/watchdog.cc

```diff
@@ -24,6 +24,7 @@
 #include "base/command_line.h"
 #include "base/json/json_reader.h"
 #include "base/json/json_writer.h"
+#include "base/logging.h"
 #include "starboard/common/file.h"
 #include "starboard/common/log.h"
```

Comment:
Nit: `#include starboard/common/log.h` can be removed too.

Reply:
Removed